modules/apps/client-extension/client-extension-service/src/main/java/com/liferay/client/extension/service/impl/ClientExtensionEntryLocalServiceImpl.java

```diff
@@ -458,6 +573,48 @@ private ClientExtensionEntry _startWorkflowInstance(
 			clientExtensionEntry, serviceContext, new HashMap<>());
 	}
 
+	private void _synchronousInvokeOnCluster(
+			Class<? extends ClusterInvokeAcceptor> clusterInvokeAcceptorClass,
+			Object targetObject, Method method, Object[] arguments,
+			long timeout, TimeUnit timeoutUnit)
+		throws Throwable {
+
```

Comment:
We might want to add this here, as a sort of sanity check:
```
		if (!ClusterExecutorUtil.isEnabled()) {
			return;
		}
```

Reply:
I did that and it doesn't work as expected. It's on by default.

But now, rethinking it, it may be that it is there so that people can temporarily disable cluster operations :thinking: 

I'll check again if it makes sense to add that. :+1:

Reply:
> I did that and it doesn't work as expected. It's on by default.

Turns out I used `ClusterInvokeThreadLocal.isEnabled()` which is a different thing, so I'll add the check you are proposing since it looks like a pattern used in the rest of the product.